docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '2'
+services:
+    mysql.local:
+        image: keybaseprivate/mysql
+        restart: unless-stopped
```

Comment:
Do we want a restart policy here? Won't that make tests less deterministic if MySQL is flaky (or is that the intent?)